boards.txt

```diff
@@ -261,6 +261,50 @@ XMC1300_Sense2GoL.menu.LIB.DSP.library.selected=-DARM_LIB_CMSIS_DSP
 #XMC1400_Boot_Kit.menu.LIB.DSPNN=ARM DSP / ARM NN Framework
 #XMC1400_Boot_Kit.menu.LIB.DSPNN.library.selected=-DARM_LIB_CMSIS_DSP -DARM_LIB_CMSIS_NN
 
+####################################################
+XMC4200_Distance2Go.name=XMC4200 Distance2Go
+XMC4200_Distance2Go.upload.tool=xmcprog
+XMC4200_Distance2Go.upload.speed=115200
+XMC4200_Distance2Go.upload.resetmethod=ck
+XMC4200_Distance2Go.upload.maximum_size=512000
+XMC4200_Distance2Go.upload.wait_for_upload_port=true
+   
+XMC4200_Distance2Go.communication=usb
+XMC4200_Distance2Go.protocol=dragon_isp
+XMC4200_Distance2Go.program.protocol=dragon_isp
+XMC4200_Distance2Go.program.tool=xmcprog
+XMC4200_Distance2Go.program.extra_params=-Pusb
+  
+XMC4200_Distance2Go.serial.disableDTR=true
+XMC4200_Distance2Go.serial.disableRTS=true
+   
+XMC4200_Distance2Go.build.mcu=cortex-m4
+XMC4200_Distance2Go.build.f_cpu=80000000L
+XMC4200_Distance2Go.build.board=ARM_XMC
+XMC4200_Distance2Go.build.board.version=4200
+XMC4200_Distance2Go.build.board.type=Q48x256
```

Comment:
If device is really a 48 pin device there are MANY pins missing in pins_arduino.h